spec/support/step_nav_steps.rb

```diff
@@ -186,4 +186,12 @@ def then_I_can_preview_the_step_by_step
   end
 
   alias_method :and_I_can_preview_the_step_by_step, :then_I_can_preview_the_step_by_step
+
+  def then_I_can_not_preview_the_step_by_step
```

Comment:
```suggestion
  def then_I_cannot_preview_the_step_by_step
```

Reply:
Thanks @brucebolt! I added your suggestion and squashed the commits.